### DISCUSSION TITLE
Any success running this on macOS?

### DISCUSSION THREAD

**Discussion by ghost** [VueTorrent/VueTorrent#371] [Q&A] [answered]
Assuming this is my own issue first.
Without this installed I do get the WebUI for qBittorrent in Safari, with it installed I don't.

Not sure where to look first for troubleshooting.
I turned off HTTPS to check... nope
I disabled all of my browser extensions... nope

The page loads successfully, there is just nothing there except the icon for VueTorrent.

<img width="691" alt="screenshot_ 2022-02-03 at 12 49 27 PM" src="https://user-images.githubusercontent.com/9438406/152409058-ea123f7a-a89b-410d-8a64-46da945f7e05.png">

**Comment by WDaan** [comment]
I don't have it installed on a Mac but I can use safari to access my hosted VueTorrent just fine.

How did you install? Native, Docker?

**Comment by ghost** [comment, marked as answer]
I just cloned the repo and added the path to qBittorrent as shown above
I've never run docker... is it necessary?

**Reply by WDaan** [reply]
No you don't need to run Docker.

Just as a sanity check...when you cloned the repo did you switch to the latest-release branch?

**Reply by ghost** [reply]
I just cloned it today and it's at master. I am not too familiar with Git... I hope that answers your question.

**Reply by ghost** [reply]
I am beginning to think it's a possible macOS Gatekeeper issue  - where apps are blocked if they are not signed?  Since it's not an app package I am not exactly sure how to verify.

**Reply by WDaan** [reply]
No worries! :)


Could you try switching to the latest-release branch and trying again?

Type `git checkout latest-release` in your terminal.
Or switch to that from github desktop, I don't know which you're using.

**Reply by ghost** [reply]
Haha!

That did it!

How could I have done that better from the get go?

I am using GitHub Desktop

Thank you.

**Reply by WDaan** [reply]
Great! I'm glad you got it working :)

You didn't do anything wrong, you just missed a step.
